syntax = "proto3";
package infos;


message HelloRequest {
     string name = 1;
}

message HelloReply {
     string message = 1;
}

// 定义一个打招呼服务
service Greeter {
    rpc SayHello(HelloRequest) returns (HelloReply) {}
    rpc SayHelloAgain(HelloRequest) returns (HelloReply) {}
}